### ISSUE TITLE
ifstat speeds were not updating.

### ISSUE THREAD

**Issue by TheBreno123p** [zzzeyez/pecan#12]
After installing ifstat and refreshing the widgets, the download and upload speeds were not updating. After some quick testing, I was able to resolve the issue.

I found that removing `-i en0` in both scripts, download and upload, resolved the issue.

The new scripts become `/usr/local/bin/ifstat -n -z -S 1 1 | awk 'FNR == 3 {print $2}'` for download and `/usr/local/bin/ifstat -n -z -S 1 1 | awk 'FNR == 3 {print $3}'` for upload.

**Comment by zzzeyez** [comment]
oh interesting.  thank you, i will fix that!

**Comment by lvadla** [comment]
For me, `/usr/local/bin/ifstat -n -z -S 1 1 | awk 'FNR == 3 {print $2}'` does **not** work but `/usr/local/bin/ifstat -i en0 -n -z -S 1 1 | awk 'FNR == 3 {print $2}'` **does** work. @zzzeyez do your scripts work with and/or without `-i en0`?

**Comment by jbbudzon** [comment]
I hate to comment in an old issue, but this seems like a reasonable place to collect information as it is tied to ifstat. I am curious if others are having issues with ifstat's "in" values when run from the terminal (not in pecan).

I am on OSX Catalina 10.15.7, performed a brew install ifstat, but running ifstat in the terminal results in the "in" column always being "0.0". The "out" column seems to update just fine. (Again, this is not an issue with pecan but seemingly ifstat.) I tried disabling the OSX firewall (and many other things) but had no success.

If anyone can confirm that ifstat works for them on Catalina, that will be helpful at least in that I will know it is something wrong with my system.

Sorry if this comment was not appropriate! I stumbled across pecan and yabai this weekend and have been setting everything up. Love it!! ❤️
